src/app/Login/page.module.css

```diff
@@ -0,0 +1,57 @@
+.EmailContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 80px;
+}
+.PasswordContainer {
+  align-items: center;
+  width: 324px;
+  height: 48px;
+  margin-bottom: 140px;
+}
+.label {
+  margin-bottom: 10px;
+  display: flex;
+}
+.span {
+  margin-top: 20px;
+  display: flex;
+  margin-bottom: 20px;
+  justify-content: center;
+}
+
+.welcomeBackcontainer {
+  width: 353px;
+  height: 45px;
+  margin-top: 150px;
+  maring-left: 19px;
+  text-align: center;
+  color: #ffffff;
+  margin-bottom: 100px;
```

Comment:
simplify code to:
```bash
margin: 100px;
text-align: center;
```

Reply:
also do not define color here or it will give issues with switching between light and dark mode